src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -225,7 +225,15 @@ private Schema buildLoadDatasetSchema() {
         "INSERT INTO `<project>.<dataset>.<stagingTable>`" +
             " (load_tag, load_time, source_name, target_path, state, file_id, checksum_crc32c, checksum_md5, error)" +
             " VALUES <load_history_array:{v|('<load_tag>', '<load_time>', '<v.sourcePath>', '<v.targetPath>'," +
-            " '<v.state>', '<v.fileId>', '<v.checksumCRC>', '<v.checksumMD5>', '<v.error>')}; separator=\",\">";
+            " '<v.state>', '<v.fileId>', '<v.checksumCRC>', '<v.checksumMD5>', \"<v.error>\")}; separator=\",\">";
+
+
+    // TODO need to test  w breaking error
```

Comment:
Two testing thoughts:
1. Make a test of the sanitizing code (to be written)
2. Used fault insertion to generate the breaking error